cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -821,7 +824,8 @@ class QueueIamSchema(IamSchema):
     """Represent the schema of IAM for Queue."""
 
     instance_profile = fields.Str(
-        metadata={"update_policy": UpdatePolicy.SUPPORTED}, validate=validate.Regexp("^arn:.*:instance-profile/")
+        metadata={"update_policy": UpdatePolicy.COMPUTE_FLEET_STOP},
```

Comment:
Should we have QUEUE_UPDATE_STRATEGY?

Reply:
no because the profile must be kept alive to be able, for the running nodes, to continue to reference it. This because during a stack update, the autogenerated profile is deleted, leaving the running nodes referencing a no-existent instance profile.